Content.Client/Chemistry/UI/ChemMasterWindow.xaml.cs

```diff
@@ -126,6 +132,17 @@ public void UpdateState(BoundUserInterfaceState state)
             PillDosage.IsValid = x => x > 0 && x <= castState.PillDosageLimit;
             BottleDosage.IsValid = x => x >= 0 && x <= bottleAmountMax;
 
+            // Begin DeltaV - chemmaster sources
+            switch (castState.Source) {
```

Comment:
use a switch expression to get the button: `var sourceButton = castState.Source switch ...`

then `sourceButton.Enabled = true;`